src/components/APOD/like_dislike_button.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
```

Comment:
hey so you've added these changes to the same branch and instead of making a new one, so we're gonna have to wait for these changes until we can merge it all in